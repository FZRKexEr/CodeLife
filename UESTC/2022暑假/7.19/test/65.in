10
eysnrcpqkdmxnyripxgwuztediwpqdtebfgrremqvsvhbhttojzbcdnscztuatzhpezureewobzcmquowhqisuprjycuoresyzlr
7
tghgyuvjjerguhchfductorqkfmyblflgmkbbbxgbpgcdrmedkusplbdbgghtazpftbvvztkisvvsbovbocxnczrklqdztemacmx
1
nqfnkylzsmoeieuvitvzwipqfiqhdueahltytejigmepkgsrdwabdliukomffsdzlydrximzdtjzbplysdgqrqklqbzivmgdztnv
3
fclkgbzmxddctmewzibhhntjebvlyhzoztfhrzoiqpctjarqynjbokdpcifpzudohqxrrecoxzpxkafghxjpzhlnwkjzfygfhnsg
2
cymttxwujbgofppvsvabyhsvhxsiqmfgedmfhkhsmwhcanrrbfgbxmuujrkmdhukaeoyhfoxhlkcprwludzxcmivmtwzhwnsjgek
3
fnfacdgctonsvmjzmnlvhnfdgltxhvqmfbtgtlmaefqantfseytnzgvwqjwihswphwzvqvqcwprczeoaasqwaotesdyfekcaqxlm
4
rbtqjxyfokmozfwvqsyinzagjubpxicaayimztcbnxnltfwbokqzqdhiacrosaftqxaapxoevdkgoyizmcrmrvwgufmhmqpgraeb
9
ifpkfwmsqesxqntqoevnljtvblegufowzotlatlaivbubysqrlgupecxnzdbxwgztenmfgreiqxykdrntiamqtkerxneavsfttwy
4
rwwnnrbmcqzjauluuqtwnfxbskbxxwkwfszqkhyfpzjeydnqqspkewqlnstpyodbkbzlzbrqfrzanmukzfrzsvvjjedovawriolw
2
ruyytmpcqwrqygosrtneofgbeivepwfcesortqbdbicxugemnwrgexvkgayzyqgruenveuyrzuxgeajzgcexkpebffoixrewylhe
4
