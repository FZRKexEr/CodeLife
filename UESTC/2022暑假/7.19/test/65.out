16384
882499718
603979768
65980984
603979768
33554432
2048
33554432
65980984
33554432
